### PR TITLE
Added Hamburger Menu

### PR DESCRIPTION
**Pull request by sergeichestakov**
Added hamburger menu button and changed up the nav bar

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+  color: grey;
+  background-color: transparent;
+  border: none;
+  font-size: 30px;
+  float: right;
+  position: fixed;
+  top: 30px;
+  right: 30px;
+}
+#mlh-trust-badge {
```
Comment (klanmiko) [inline css/styles.css:25]: thanks for moving the trust badge css out of inline!

File: index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```
Comment (klanmiko) [inline index.html:9]: this is already in the index.html file above

File: css/styles.css (hunk 1 above)
Comment (sergeichestakov) [reply css/styles.css:25]: No problem bud!

File: index.html (hunk 2 above)
Comment (sergeichestakov) [reply index.html:9]: Yeah, but it wasn't working. Maybe we should remove all the scripts and css were not using cuz I'm pretty sure a bunch of them don't work.

**Pull request merged**